students/smitco/lesson03/mailroom.py

```diff
@@ -44,25 +45,43 @@ def intro():
     else:
         return addressee    
 def donation(n):
-    dollars = input("What is the donation amount? ")
-    if int(dollars) < (10 ** 6):
-        position = donors.index(n)
-        a = amounts[position]
-        a = a + (int(dollars),)
-        amounts[position] = a
-        return dollars
+    if n == "quit":
+        print()
+        pass
     else:
-        print("\nNice try, Moneybags.")
-        quit()
+        dollars = input("What is the donation amount? ")
+        if dollars == "quit":
+            donors.remove(n)
+            print()
+            pass
+        elif int(dollars) < (10 ** 6):
+            position = donors.index(n)
+            a = amounts[position]
+            a = a + (int(dollars),)
+            amounts[position] = a
+            return dollars
+        else:
+            print("\nNice try, Moneybags.\n")
+            quit()
 def card(n, v):
```

Comment:
avoid using `n` and `v` type of vars since the reader won't know what those mean.

Reply:
Thanks! I'm trying to break this bad habit.